File: 2894. Divisible and Non-divisible Sums Difference.py
class Solution:
    def differenceOfSums(self, n: int, m: int) -> int:
        num1 = 0
        num2 = 0
        for x in range (1, n+1):
            if (x % m == 0):
                num2 += x
            else:
                num1 += x
        return num1 - num2

File: 2894. Divisible and Non-divisible Sums Difference.js
/**
 * @param {number} n
 * @param {number} m
 * @return {number}
 */
var differenceOfSums = function(n, m) {
    num1 = 0;
    num2 = 0;
    for (let i = 1; i <= n; i++){
        if (i % m == 0) num2 += i;
        else num1 += i;
    }
    return num1 - num2; 
};